File: examples/nodejs/console-url.js
var request = require('request');
var options = {
  'method': 'POST',
  'url': 'https://api.novoserve.com/v0/servers/000-000/ipmi-link',
  'headers': {
    'Authorization': 'Basic [BASE64 STRING OF KEY+SECRET]',
    'Content-Type': 'application/json'
  },
  body: JSON.stringify({
    "remoteIp": "127.0.0.1",
    "whitelabel": "yes"
  })

};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});

File: examples/ruby/console-url.rb
require "uri"
require "json"
require "net/http"

url = URI("https://api.novoserve.com/v0/servers/000-000/ipmi-link")

https = Net::HTTP.new(url.host, url.port)
https.use_ssl = true

request = Net::HTTP::Post.new(url)
request["Authorization"] = "Basic [BASE64 STRING OF KEY+SECRET]"
request["Content-Type"] = "application/json"
request.body = JSON.dump({
  "remoteIp": "127.0.0.1",
  "whitelabel": "yes"
})

response = https.request(request)
puts response.read_body
